data/service/api/v1/alerts.go

```diff
@@ -0,0 +1,112 @@
+package v1
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"log"
+
+	"github.com/tidepool-org/platform/alerts"
+	"github.com/tidepool-org/platform/data/service"
+	"github.com/tidepool-org/platform/permission"
+	"github.com/tidepool-org/platform/request"
+	platform "github.com/tidepool-org/platform/service"
+)
+
+func AlertsRoutes() []service.Route {
+	return []service.Route{
```

Comment:
Is it possible to expose a `GET` route for fetching an invite by id? This way it can be used by the QA team for testing purposes.

Reply:
Absolutely, though I was planning for a follow-on PR for that, as this one is already quite large.

Reply:
To clarify, I will add the GET route now, for easier testing in QA.